### ISSUE TITLE
Berryboot breaks if you put your USB drive to PC and back

### ISSUE THREAD

**Issue by seppy87** [maxnet/berryboot#640]
Hallo,

I wanted to make a backup of my USB Drive. I put it in my PC and made a Backup. Afterwards it says it could not find data Partition.

Now I tried it again. But now I did not do anything except connecting the working Berryboot USB Drive to my PC and back to Raspberry (same slot). Now it says can't find data Partition too.

I am getting annoyed.
Any Fix or any alternative to Berryboot who is more userfriendly?

**Comment by maxnet** [comment]
Did your PC by any chance offer to format the data partition?
That is the only reason I can think of that can and will destroy your data.

**Comment by seppy87** [comment]
I don't think so. every question to format the disk was answered with no/cancel. And Paragon Linux FS for Windows is showing that the Linux Partition is still there.

**Comment by gotouchgo** [comment]
Well, I actually hit similar SD card issue. After I installed the OS using my berryboot usb drive, the date of cmdline.txt and wpa_supplicant.conf are set to 1980/01/01, (1979/12/31 when look at in Windows), Then I reboot the OS, SD card still plugged in, but it can't boot at all. I have to reformat the SD card on Windows, and copy over all berryboot files from the zip, then it can boot, but I have to install OS again.

So I backup the files, reformat the SD card on Windows, copy over all berryboot files from the zip, open the cmdline.txt and wpa_supplicant.conf with notepad and save with current time. Then copy these two files to the newly formatted SD card. Now finally it boots up correctly. It is so weird.

Additionally I see a small partition of 127MB is created on my USB SSD drive with a copy of berryboot files there. My USB SSD is bootable, but not sure if possible to use those files to boot.
